src/Microsoft.Azure.WebJobs.Host/Singleton/SingletonManager.cs

```diff
@@ -286,6 +281,15 @@ public static SingletonAttribute GetListenerSingletonOrNull(Type listenerType, F
             return singletonAttribute;
         }
 
+        internal static TimeSpan GetLockAcquisitionTimeout(SingletonAttribute attribute, SingletonConfiguration config)
+        {
+            TimeSpan acquisitionTimeout = attribute.LockAcquisitionTimeout != -1
+                    ? TimeSpan.FromSeconds(attribute.LockAcquisitionTimeout) :
```

Comment:
nit: I know this code just moved and this is how it was before, but can you move the `:` to the line below? :)